scripts/README.rst

```diff
@@ -91,34 +91,32 @@ Assuming you have the two prerequisites mentioned above, install the
 dependencies for the report generator, and run it, passing the following
 values on the command line:
 
-    * feature-toggle-data: path to the sql dump data created above
-    * annotation-data: path to the code annotation data created above
-    * reports: path to write report files into
-    * environment: name of the environment/deployment that you are reporting on
-    * --publish: (optional) a flag to specify whether or not to publish
-      the resulting HTML report to Confluence
 
 For example:
 
 .. code:: bash
 
-    python -m scripts.feature_toggle_report_generator my_data my_annotations output_dir stage --publish
+    python -m scripts.feature_toggle_report_generator annotation_dir_path toggle_data_dir_path output_path --show-state
 
-NOTE: If you choose to publish to Confluence, you must have the following
-environment variables set to be able to do so:
 
-* CONFLUENCE_BASE_URL: the url of the confluence instance you are targeting. For
-  example: https://my-company.atlassian.net
-* CONFLUENCE_API_TOKEN: a token for accessing the confluence api
-* CONFLUENCE_USER_EMAIL: the email address of the user linked to the api token
-* CONFLUENCE_SPACE_ID: the id of the space in confluence where you will publish the report
-* CONFLUENCE_PAGE_NAME: the name of the page that will host your report. If it is not yet
-  created, this tool will create it.
 
 .. code:: bash
 
     make requirements
-    python scripts/feature_toggle_report_generator.py feature-toggle-data annotation-data reports environment
+    python -m scripts.feature_toggle_report_generator annotation_dir_path toggle_data_dir_path output_path --show-state
```

Comment:
For consideration of a future iteration, are we ever planning on using this for anything other than outputting state?  If not, we may want to consider one day renaming to `generate_feature_toggle_state_report` and dropping the argument and related conditionals.